src/ralph/backends/http/lrs_async.py

```diff
@@ -0,0 +1,112 @@
+"""LRS HTTP backend for Ralph."""
+
+import logging
+import json
+from typing import Iterable, Union, Iterator
+from urllib.parse import urlparse
+from more_itertools import chunked
+
+from httpx import AsyncClient, Headers
+from pydantic import HttpUrl
+
+from ralph.conf import settings
+from ralph.exceptions import BackendException
+
+from .base import BaseHTTP
+
+#lrs_settings = settings.BACKENDS.HTTP.LRS
+logger = logging.getLogger(__name__)
+
+
+class LRSHTTP(BaseHTTP):
+    """LRS HTTP backend."""
+    name = "lrs"
```

Comment:
As we expect to have both a synchronous and asynchronous backends, I would suggest renaming this backend accordingly:

```suggestion
class AsyncLRSHTTP(BaseHTTP):
    """LRS HTTP backend."""
    name = "alrs"
```